demos/projects/ESPRESSIF/aziotkit/components/azure-iot-kit-sensors/src/oled.c

```diff
@@ -0,0 +1,45 @@
+/* OLED screen Example
+
+   For other examples please check:
+   https://github.com/espressif/esp-iot-solution/tree/master/examples
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+#include <stdio.h>
+#include <time.h>
+#include <sys/time.h>
+#include "oled.h"
+
+void oled_show_signs(ssd1306_handle_t dev);

```

Comment:
>oled_show_signs [](http://example.com/codeflow?start=5&length=15)

This is not part of include where are we using it? Can we reduce the scope by making it static?